Include/internal/pycore_bitutils.h

```diff
@@ -49,7 +49,7 @@ _Py_bswap16(uint16_t word)
 static inline uint32_t
 _Py_bswap32(uint32_t word)
 {
-#ifdef _PY_HAVE_BUILTIN_BSWAP
+#if _Py_has_builtin(__builtin_bswap32) || defined(_PY_HAVE_BUILTIN_BSWAP)
```

Comment:
I have the feeling that `_Py_has_builtin(__builtin_bswap32)` should be used to correctly position `_PY_HAVE_BUILTIN_BSWAP` instead of being used here. That way if  `_PY_HAVE_BUILTIN_BSWAP` is mentioned elsewhere, it will have the correct value.

Reply:
_PY_HAVE_BUILTIN_BSWAP is used to get the optimization on GCC older than GCC 10. __has_feature() was added very recently in GCC: in GCC 10.